IPython/html/base/handlers.py

```diff
@@ -138,6 +138,10 @@ def base_url(self):
         return self.settings.get('base_url', '/')
 
     @property
+    def default_url(self):
+        return self.settings.get('default_url', '')
```

Comment:
There are no redirects associated with this value. An href to '' doesn't go anywhere.